test_common/harness/kernelHelpers.cpp

```diff
@@ -665,11 +665,42 @@ static int create_single_kernel_helper_create_program_offline(cl_context context
 
         size_t length = modifiedKernelBuf.size();
         log_info("offlineCompiler: clCreateProgramWithSource replaced with clCreateProgramWithIL\n");
+        if (gCoreILProgram)
+        {
+            *outProgram = clCreateProgramWithIL(context, &modifiedKernelBuf[0],
+                                                length, &error);
+        }
+        else
+        {
+            cl_platform_id platform;
+            error = clGetDeviceInfo(device, CL_DEVICE_PLATFORM,
+                                    sizeof(cl_platform_id), &platform, NULL);
+            print_error(error, "clGetDeviceInfo for CL_DEVICE_PLATFORM failed");
```

Comment:
```suggestion
            if(error != CL_SUCCESS)
            {
                print_error(error, "clGetDeviceInfo for CL_DEVICE_PLATFORM failed");
            }
```

Reply:
Ok I've used test_error which include return condition in case of error != CL_SUCCESS